### ISSUE TITLE
Accessibility: Application Menu Personalized Roles extra tab or missing tab indicator 

### ISSUE THREAD

**Issue by CindyMercadoReyes** [infor-design/enterprise#2327]
**Describe the bug**
Application Menu Personalized Roles: Accessibility Issue with tab key 

**To Reproduce**
1. Go to url: http://4190-beta2-enterprise.demo.design.infor.com/components/applicationmenu/example-personalized-roles.html
2. On the application menu
3. If Employee, Manager, or Recruiter are collapsed and I tab key to go from Employee to Manager
4. I have to tab key twice in order for the field box to be highlighted
5. The error here is that I should not have to tab key twice in order to move to the next field below

**Expected behavior**
The error here is that I should not have to tab key twice in order to move to the next field below

**Version**

- ids-enterprise: [e.g. v4.9.0 or v4.10.0]

**Screenshots**
<img width="500" alt="application menu" src="https://user-images.githubusercontent.com/39101346/59048552-0be99a80-8854-11e9-8e35-2c2e7ac30227.png">

**Platform**
 - OS Version: [iOS 10.14.4]
 - Browser Name [Chrome]

**Additional context**
None

**Comment by tmcconechy** [comment]
There are sort of two issues here..

If employee is focused and collapsed:
- tab once will tab you to the chevron but whats the point? as space will toggle it when either the header or the chevron is focused
- but WORSE you have to then keep tabbing and its hitting all the hidden elements

**Comment by EdwardCoyle** [comment]
FWIW, the reasoning for the double tab stop is because it's possible for the hyperlink in Accordion Headers to be an outbound link.  It's arguable that in the case of it being empty, there should be only one tab stop.

However, when you use the arrow keys inside an accordion, it should navigate up/down the headers with one keystroke instead of two.

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
